pkg/apply/taskrunner/context.go

```diff
@@ -53,6 +53,12 @@ func (tc *TaskContext) ResourceCache() cache.ResourceCache {
 	return tc.resourceCache
 }
 
+// SendEvent sends an event on the event channel
+func (tc *TaskContext) SendEvent(e event.Event) {
+	klog.V(5).Infof("sending event: %s", e)
```

Comment:
I'd make this debug level. Events are usually consumed by printers, so there is no need to log it and then also print it.

Reply:
AFAICT cli-utils has no debug or even warning logging. It's all info and error logs with various levels of verbosity.

It's not really clear to me when to use each type with klog or even really what the levels are intended for. But if we want to standardize it should probably be a different cleanup PR.